pytorch_lightning/loggers/base.py

```diff
@@ -349,6 +350,49 @@ def _add_prefix(self, metrics: Dict[str, float]):
 
         return metrics
 
+    def _scan_checkpoints(self, checkpoint_callback: "ReferenceType[ModelCheckpoint]") -> List[ModelCheckpoint]:
```

Comment:
I totally agree on this, since the function is only used by two of the loggers. Where should I put it? In `base.py`, outside the `LightningLoggerBase` class?

What about the other two functions:
- `_scan_and_log_checkpoints(...)`
- `_log_checkpoints(...)`

They are also only used by those two loggers, should I also remove them from the base class?